src/picosvg/svg_pathops.py

```diff
@@ -183,6 +183,7 @@ def stroke(
 
     # nuke any conics that snuck in (e.g. with stroke-linecap="round")
     sk_path.convertConicsToQuads(tolerance)
+    sk_path.simplify(fix_winding=True)
 
     assert sk_path.fillType == pathops.FillType.WINDING
```

Comment:
you can now remove this assert, since calling simplify with fix_winding=True guarantees the fillType is going to be WINDING anyway

Reply:
OK

Reply:
Done.